src/ApiSummary.js

```diff
@@ -331,8 +336,8 @@ export class ApiSummary extends AmfHelperMixin(LitElement) {
       return '';
     }
     return html`
-    <div class="api-title" role="heading" aria-level="${titleLevel}">
-    <label>API title:</label>
+    <div class="api-title" role="heading" aria-level="${titleLevel}" part="api-title">
+    <label part="api-title-label">API title:</label>
```

Comment:
Do we need to include the CSS Parts in the doc so people know what is capable?

Reply:
That's a good idea, I'll include a doc